app.py

```diff
@@ -1,44 +1,62 @@
+# app-puakaldi-wrapper version 0.1.0
+# author: Angus L'Herrou
+# org: CLAMS team
+
 import json
-import os, shutil
+import os
+import shutil
 import subprocess
-from typing import Dict
+from typing import Dict, Sequence, Tuple, List, Union
+import argparse
 
 from clams import ClamsApp, Restifier
 from mmif import Mmif, View, Annotation, Document, AnnotationTypes, DocumentTypes, Text
+from lapps.discriminators import Uri
 
-KALDI_MEDIA_DIRECTORY = '/audio-in'
-KALDI_16KHZ_DIRECTORY = '/audio_in_16khz'
-KALDI_EXPERIMENT_DIR = '/kaldi/egs/american-archive-kaldi/sample_experiment'
+MEDIA_DIRECTORY = '/audio_in'
+MEDIA_DIRECTORY_16KHZ = '/audio_in_16khz'
+KALDI_AMERICAN_ARCHIVE = '/kaldi/egs/american-archive-kaldi'
+KALDI_EXPERIMENT_DIR = os.path.join(KALDI_AMERICAN_ARCHIVE, 'sample_experiment')
 KALDI_OUTPUT_DIR = os.path.join(KALDI_EXPERIMENT_DIR, 'output')
-KALDI_VERSION = 'IDK'
+TRANSCRIPT_DIR = os.path.join(MEDIA_DIRECTORY, 'transcripts')
+KALDI_VERSION = '0.1.0'
```

Comment:
I'd like to sugest rename this to `APP_VERSION` .

Reply:
Good idea.